pom.xml

```diff
@@ -48,20 +43,23 @@
       <artifactId>log4j-core</artifactId>
       <version>2.20.0</version>
     </dependency>
+      <dependency>
+          <groupId>org.mockito</groupId>
+          <artifactId>mockito-core</artifactId>
+          <version>5.3.1</version>
```

Comment:
Might exist a newer version of [mockito](https://mvnrepository.com/artifact/org.mockito/mockito-core)?

Reply:
Will look in to that

Reply:
Latest push included update of mockito version